src/gt4py/next/program_processors/runners/dace/transformations/map_fusion_extended.py

```diff
@@ -459,6 +460,27 @@ def can_be_applied(
 
         # TODO(phimuell): Implement a check that the array is single use data?
 
+        # Check that the data read by the nested map are not accessed by any other outgoing edge
+        # of the access node.
+        access_node_out_edges = graph.out_edges(self.access_node)
```

Comment:
```suggestion
        access_node_out_edges = list(graph.out_edges(self.access_node))
```
I am not sure if it will always be a `list` or if they will change it at some point to an iterator.